README.md

```diff
@@ -172,6 +172,8 @@ print(Mem.cell1[63])
 
 > **Note**: if you are using function calls, *pyndustric* will use `cell1` as a call stack, so you might not want to use `cell1` in that case to store data in it. (If you aren't calling any functions in your code, using `cell1` should be fine.)
 
+> Alternatively, you can mark your functions with the `@inline` decorator, and it will compile them *inline*, so the function code gets copied to each function call. It will make it less efficient, but allows you to not need a separate memory cell.
```

Comment:
Since there's still conflicts to resolve, I'll nitpick more.

> It will make it less efficient

As with many things development, this is all about tradeoffs. If it was a function extracted to make some code easier to read, but is actually only used once, it will save quite a bit of overhead. But as soon as it's used multiple times, it could quickly bloat code size indeed.

```suggestion
> Alternatively, you can mark your functions with the `@inline` decorator, and it will compile them *inline*, so the function code gets copied to each function call. This is faster and means you don't need a memory cell, but if the function is used more than once, it will quickly bloat the generated code size.
```